<ui version="4.0" >
 <class>TabBarConfigWidget</class>
 <widget class="QWidget" name="TabBarConfigWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>465</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="gbBehaviour" >
     <property name="title" >
      <string>Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <widget class="QLabel" name="lblLocation" >
        <property name="text" >
         <string>Location:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cmbLocation" >
        <item>
         <property name="text" >
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bottom</string>
         </property>
        </item>
       </widget>
      </item>
      <item rowspan="2" row="0" column="2" >
       <spacer name="horizontalSpacer_3" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>98</width>
          <height>53</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lblRows" >
        <property name="text" >
         <string>Rows:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="sbRows" >
        <property name="suffix" >
         <string> rows</string>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lblSorting" >
        <property name="text" >
         <string>Sorting:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QCheckBox" name="chkFollowActive" >
          <property name="text" >
           <string>Keep activated tab visible</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="1" colspan="2" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QComboBox" name="cmbSorting" >
          <item>
           <property name="text" >
            <string>Opening Order</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Document Name</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Document URL</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>File Extension</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QGroupBox" name="gbTabs" >
     <property name="title" >
      <string>Tabs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="1" column="0" >
       <widget class="QLabel" name="lblMinWidth" >
        <property name="text" >
         <string>Minimum width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="sbMinWidth" >
        <property name="suffix" >
         <string> pixels</string>
        </property>
        <property name="minimum" >
         <number>20</number>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="1" column="2" >
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>11</width>
          <height>96</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="lblMaxWidth" >
        <property name="text" >
         <string>Maximum width:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QSpinBox" name="sbMaxWidth" >
        <property name="text" stdset="0" >
         <string>20 pixels</string>
        </property>
        <property name="suffix" >
         <string> pixels</string>
        </property>
        <property name="minimum" >
         <number>20</number>
        </property>
        <property name="maximum" >
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="lblHeight" >
        <property name="text" >
         <string>Height:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="sbHeight" >
        <property name="text" stdset="0" >
         <string>16 pixels</string>
        </property>
        <property name="suffix" >
         <string> pixels</string>
        </property>
        <property name="minimum" >
         <number>16</number>
        </property>
        <property name="maximum" >
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblStyle" >
        <property name="text" >
         <string>Style:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="cmbStyle" >
        <item>
         <property name="text" >
          <string>Buttons</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Flat</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="gbHighlighting" >
     <property name="title" >
      <string>Highlighting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <layout class="QGridLayout" name="gridLayout_3" >
          <item row="0" column="0" >
           <widget class="KColorButton" name="colModified" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="chkModified" >
            <property name="text" >
             <string>Highlight modified tabs</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="KColorButton" name="colActive" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="chkActive" >
            <property name="text" >
             <string>Highlight active tab</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="KColorButton" name="colPrevious" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QCheckBox" name="chkPrevious" >
            <property name="text" >
             <string>Highlight previous tab</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="lblOpacity" >
            <property name="text" >
             <string>Opacity:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QSlider" name="slOpacity" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_5" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>241</width>
            <height>108</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QLabel" name="lblNote" >
          <property name="text" >
           <string>Note: Use the context menu to highlight a tab</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>0</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearCache" >
          <property name="whatsThis" >
           <string>Remove all highlight marks in the current session.</string>
          </property>
          <property name="text" >
           <string>Clear Highlight Cache</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="gbPreview" >
     <property name="title" >
      <string>Preview</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbLocation</tabstop>
  <tabstop>sbRows</tabstop>
  <tabstop>chkFollowActive</tabstop>
  <tabstop>cmbSorting</tabstop>
  <tabstop>cmbStyle</tabstop>
  <tabstop>sbMinWidth</tabstop>
  <tabstop>sbMaxWidth</tabstop>
  <tabstop>sbHeight</tabstop>
  <tabstop>chkModified</tabstop>
  <tabstop>colModified</tabstop>
  <tabstop>chkActive</tabstop>
  <tabstop>colActive</tabstop>
  <tabstop>chkPrevious</tabstop>
  <tabstop>colPrevious</tabstop>
  <tabstop>slOpacity</tabstop>
  <tabstop>btnClearCache</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkModified</sender>
   <signal>toggled(bool)</signal>
   <receiver>colModified</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>111</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel" >
     <x>50</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkActive</sender>
   <signal>toggled(bool)</signal>
   <receiver>colActive</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>96</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel" >
     <x>55</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPrevious</sender>
   <signal>toggled(bool)</signal>
   <receiver>colPrevious</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>84</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
